modules/standard/Math.chpl

```diff
@@ -1048,6 +1048,72 @@ module Math {
     return chpl_gcd(x, y);
   }
 
+  /* Returns the greatest common divisor of the integer arguments `x` and
+     `y`. */
```

Comment:
Not sure how to handle it, but duplicating the docs for each of these functions seems not ideal. Could/should we make some no-doc'd? Or maybe shorten the doc? Mentioning `x` and `y` does not seem particularly helpful.

Reply:
Mentioning the arguments directly was recently added by an outside contributor, iirc.

Reply:
Back on the duplication of docs - I don't mind these ones so much because they are a single line.  There's not much shorter we can go on that.

Reply:
If you time, it'd be good to make an issue to bring up the problem / request some improvement to chpldoc to make it easier to document a family of overloads.

Reply:
I opened #23353, feel free to add anything you think is missing there :)